app/javascript/components/userinput.vue

```diff
@@ -26,9 +26,7 @@
       },
       value: {
         type: Object,
-        default: {
-          id: 0
-        }
+        default: {}
```

Comment:
Did you check this work on other places where the userinput is used?

Reply:
Yes, it is only used on the user credit mutations and that still works as expected